parsec/core/fs/workspacefs/workspacefs.py

```diff
@@ -252,7 +261,9 @@ def is_read_only(self) -> bool:
     def is_revoked(self) -> bool:
         return self.get_workspace_entry().role is None
 
-    async def path_info(self, path: AnyPath) -> Dict[str, object]:
+    # Information
```

Comment:
Useless comment, the function name is explicit enough.

Reply:
done